app-service-template/res/configuration.toml

```diff
@@ -27,11 +27,11 @@ LogLevel = "INFO"
     [Writable.Telemetry.Metrics] # All service's metric names must be present in this list.
     MessagesReceived = true
     InvalidMessagesReceived = true
-    PipelineMessagesProcessed = true # Your pipeline IDs are added to this name for the actual metric name reported
-    PipelineMessageProcessingTime = true # Your pipeline IDs are added to this name for the actual metric name reported
-    PipelineProcessingErrors = true # Your pipeline IDs are added to this name for the actual metric name reported
-    HttpExportSizeName = true
-    MqttExportSizeName = true
+    PipelineMessagesProcessed = true # Your pipeline IDs are added as tag for the metric for each pipeline defined
```

Comment:
```suggestion
    PipelineMessagesProcessed = true # Your pipeline IDs are added as the tag for the metric for each pipeline defined
```

Reply:
Do we want this to be in second person with `Your` or should we just say - `pipeline Ids are added as the tag for`...

Reply:
Fixing.